### PR TITLE
[WIP]RDBMS slot information database tables

### PR DESCRIPTION
**Pull request by AnujaLK**
RDBMS slot information database tables

### CODE REVIEW

File: features/andes/org.wso2.carbon.andes.server.feature/resources/dbscripts/mssql-mb.sql [hunk 1]
```diff
@@ -113,4 +113,37 @@ CREATE TABLE MB_QUEUE_COUNTER (
                         PRIMARY KEY (QUEUE_NAME)      
 );
 
+IF NOT EXISTS (SELECT * FROM sys.objects WHERE OBJECT_ID = OBJECT_ID(N'[DB0].[MB_SLOT]') AND TYPE IN (N'U'))
+CREATE TABLE MB_SLOT (
+                        SLOT_ID bigint IDENTITY(1,1) NOT NULL,
+                        START_MESSAGE_ID bigint NOT NULL,
+                        END_MESSAGE_ID bigint NOT NULL,
+                        STORAGE_QUEUE_NAME varchar(100) NOT NULL,
+                        SLOT_STATE tinyint NOT NULL DEFAULT '1',
```
Comment (ramith) [inline features/andes/org.wso2.carbon.andes.server.feature/resources/dbscripts/mssql-mb.sql]: Let's put a comment explaining what '1' stands for. ( slotstate)

**Comment by pumudu88** [comment]
Please point to related jira's (if there's any) in the description.

**Comment by AnujaLK** [comment]
RDBMS Slot storing feature - https://wso2.org/jira/browse/MB-1149

File: features/andes/org.wso2.carbon.andes.server.feature/resources/dbscripts/mysql-mb.sql [hunk 2]
```diff
@@ -101,6 +101,37 @@ CREATE TABLE IF NOT EXISTS MB_QUEUE_COUNTER (
...
+                        QUEUE_NAME varchar(512) NOT NULL,
+                        MESSAGE_ID bigint(20) NOT NULL,
+                        PRIMARY KEY (QUEUE_NAME,MESSAGE_ID)
+) ENGINE=MyISAM DEFAULT CHARSET=latin1;
+
+CREATE TABLE IF NOT EXISTS MB_NODE_TO_LAST_PUBLISHED_ID (
+                        NODE_ID varchar(512) NOT NULL,
+                        MESSAGE_ID bigint(20) NOT NULL,
+                        PRIMARY KEY (NODE_ID)
+) ENGINE=MyISAM DEFAULT CHARSET=latin1;
```
Comment (ramith) [inline features/andes/org.wso2.carbon.andes.server.feature/resources/dbscripts/mysql-mb.sql]: shall we switch to innodb?

File: features/andes/org.wso2.carbon.andes.server.feature/resources/dbscripts/mssql-mb.sql (hunk 1 above)
Comment (AnujaLK) [inline features/andes/org.wso2.carbon.andes.server.feature/resources/dbscripts/mssql-mb.sql]: Fixed with PR https://github.com/wso2/carbon-business-messaging/pull/118

File: features/andes/org.wso2.carbon.andes.server.feature/resources/dbscripts/mysql-mb.sql (hunk 2 above)
Comment (AnujaLK) [inline features/andes/org.wso2.carbon.andes.server.feature/resources/dbscripts/mysql-mb.sql]: Fixed with PR https://github.com/wso2/carbon-business-messaging/pull/118

**Pull request merged**